src/oc_zipkin_reporter.erl

```diff
@@ -0,0 +1,101 @@
+%%%------------------------------------------------------------------------
+%% Copyright 2018, OpenCensus Authors
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%% http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+%%
+%% @doc Exports spans to Zipkin
+%% @end
+%%%------------------------------------------------------------------------
+
+
+-module(oc_zipkin_reporter).
+
+-include("opencensus.hrl").
+
+-behaviour(oc_reporter).
+
+-define(DEFAULT_ZIPKIN_ADDRESS, "http://localhost:9411").
+-define(DEFAULT_LOCAL_ENDPOINT, #{<<"serviceName">> => node()}).
+
+-export([init/1,
+         report/2]).
+
+init(Opts) ->
+    Opts.
+
+report(Spans, _Opts) ->
+    Address = zipkin_address(),
+    ZSpans = [zipkin_span(Span) || Span <- Spans],
+
+    try jsx:encode(ZSpans) of
+        JSON ->
+            case httpc:request(post,
+                               {Address ++ "/api/v2/spans", [],
```

Comment:
Not a big deal but could also put the full combined URL in Opts instead of having to create a new list every time report is called.

Reply:
+1